config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_bigquerydatasets.bigquery.cnrm.cloud.google.com.yaml

```diff
@@ -16,10 +16,8 @@ spec:
     categories:
     - gcp
     kind: BigQueryDataset
+    listKind: BigQueryDatasetList
     plural: bigquerydatasets
-    shortNames:
-    - gcpbigquerydataset
-    - gcpbigquerydatasets
```

Comment:
You need to update the go tag in apis to include these shortnames, example https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/8fe8c6dd64cc1034dfb2ab00105bf258dc885406/apis/kms/v1beta1/kmskeyring.go#L71